fluid/ExternalCodeEditor_WIN32.cxx

```diff
@@ -312,9 +337,12 @@ int ExternalCodeEditor::remove_tmpfile() {
 //     Returns pointer to static memory.
 //
 const char* ExternalCodeEditor::tmpdir_name() {
-  char tempdir[100];
-  if (GetTempPath(sizeof(tempdir), tempdir) == 0 ) {
+  wchar_t tempdirW[FL_PATH_MAX+1];
+  char tempdir[FL_PATH_MAX+1];
+  if (GetTempPathW(FL_PATH_MAX, tempdirW) == 0) { // FIXME: word
```

Comment:
What does this comment "FIXME: word" mean? It should have an obvious meaning to the reader, otherwise it's useless and confusing. Like it confuses me. ;-)

Reply:
"Fix Me".. like TODO or XXX

Reply:
A FIXME is similar to a TODO in that it marks a point in the code that still needs to be looked at. TODOs are stuff, that the programmer wants to come back to eventually, to improve thing or add features. FIXMEs however mark bugs that could not be fixed right now, but must be fixed as soon as possible. Some editors and code quality tools recognise TODO: and FIXME: and highlight them. Xcode inserts a band-aid icon in the function list pulldown.

In this case, I marked all the functions that needed to be converted to wide characters before actually changing them. This makes it very easy to find the lines in Xcode. I simply forgot to remove that last one. It's removed now.

Reply:
@erco77 @MatthiasWM Thanks for all the kind explanations, but this was not my question. I know what "FIXME" means, it was about this particular text: "word". But Matthias explained this (and removed it now), thanks.

I didn't know that Xcode flags **comments**, that's a nice feature (but I'm not using Xcode).